static/main.js

```diff
@@ -87,7 +100,7 @@ window.onload = function() {
 
 
     socket.on("update", function(msg) {
-        if (!window.gameBegin) {
+        if (!window.gameBegin || window.bike === undefined) {
```

Comment:
I'm not as good at JS, but i think it is better to use typeof to check if object is undefined 
```suggestion
        if (!window.gameBegin || typeof window.bike !== "undefined") {
```